### PR TITLE
fix(DB/Core): Freya - Sun beam

### PR DESCRIPTION
**Pull request by Si1ker**
These changes will fix the Sun beam event on 25-man raid and place the correct models into the dummies which are triggering the spell.

In the current Freya event at AC, Freya is summoning just 1 sun beam for 10 and 25 mode, which is wrong, this needs to be 1 for 10-man mode and 3 for 25-man mode.

Reference: https://www.youtube.com/watch?v=7wxv88pA3Ew  (3:08)

## CHANGES PROPOSED:
-  Fix models on dummies which are triggering the spell
-  Add to more summons on 25-man raid

## ISSUES ADDRESSED:
- Closes 

## TESTS PERFORMED:
- Tested in-game
image

## HOW TO TEST THE CHANGES:

Change raid mode to 25-man
.go c Freya
Engage combat and wait for the three Sun Beams positions

## KNOWN ISSUES AND TODO LIST:

Freya damage WTF  D :
- [ ] #3552 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by aradep** [comment]
Getting an error during build. Win 10 + Docker
image

**Comment by Si1ker** [comment]
Ty @aradep , will be sorted out tomorrow, going to bed now 🐱

File: src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp [hunk 1]
```diff
@@ -644,6 +644,19 @@ class boss_freya : public CreatureScript
...
                     cr->CastSpell(cr, SPELL_UNSTABLE_SUN_FREYA_DAMAGE, true);
                 }
+                if (Is25ManRaid())
+                {
+                    if (Creature* cr = me->SummonCreature(NPC_FREYA_UNSTABLE_SUN_BEAM, me->GetPositionX()+urand(7,25), me->GetPositionY()+urand(7,25), me->GetMap()->GetHeight(me->GetPositionX(), me->GetPositionY(), MAX_HEIGHT), 0, TEMPSUMMON_TIMED_DESPAWN, 10000))
+                    {
+                        cr->CastSpell(cr, SPELL_UNSTABLE_SUN_VISUAL, true);
+                        cr->CastSpell(cr, SPELL_UNSTABLE_SUN_FREYA_DAMAGE, true);
+                    }
+                    if (Creature* cr = me->SummonCreature(NPC_FREYA_UNSTABLE_SUN_BEAM, me->GetPositionX()+urand(7,25), me->GetPositionY()+urand(7,25), me->GetMap()->GetHeight(me->GetPositionX(), me->GetPositionY(), MAX_HEIGHT), 0, TEMPSUMMON_TIMED_DESPAWN, 10000))
```
Comment (FrancescoBorzi) [inline src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp:654]: why twice the same `if` ?

Comment (Winfidonarleyan) [reply src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp:654]: I was wondering, too.

Comment (Kitzunu) [reply src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp:654]: Well there are three of the same if

Comment (FrancescoBorzi) [reply src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp:654]: @Si1ker

File: src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp [hunk 2]
```diff
@@ -639,23 +641,11 @@ class boss_freya : public CreatureScript
                 events.RepeatEvent(45000+urand(0,10000));
                 break;
             case EVENT_FREYA_UNSTABLE_SUN_BEAM:
-                if (Creature* cr = me->SummonCreature(NPC_FREYA_UNSTABLE_SUN_BEAM, me->GetPositionX()+urand(7,25), me->GetPositionY()+urand(7,25), me->GetMap()->GetHeight(me->GetPositionX(), me->GetPositionY(), MAX_HEIGHT), 0, TEMPSUMMON_TIMED_DESPAWN, 10000))
-                {
-                    cr->CastSpell(cr, SPELL_UNSTABLE_SUN_VISUAL, true);
-                    cr->CastSpell(cr, SPELL_UNSTABLE_SUN_FREYA_DAMAGE, true);
-                }
+                me->SummonCreature(NPC_FREYA_UNSTABLE_SUN_BEAM, me->GetPositionX()+urand(7,25), me->GetPositionY()+urand(7,25), me->GetMap()->GetHeight(me->GetPositionX(), me->GetPositionY(), MAX_HEIGHT), 0, TEMPSUMMON_TIMED_DESPAWN, 10000);
```
Comment (Kitzunu) [inline src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp:644]: ?

**Comment by Si1ker** [comment]
@aradep test again if u can.

**Comment by FrancescoBorzi** [comment]
@Si1ker it would be better to solve the conflicts first

**Comment by Si1ker** [comment]
Will re-do this weekend 🐱

**Pull request closed without merging**